### ISSUE TITLE
[FORMATTING] Mysql ON DUPLICATE KEY UPDATE not format as expected

### ISSUE THREAD

**Issue by xiaoxiyao** [sql-formatter-org/sql-formatter#605]
**Input data**

Which SQL and options did you provide as input?

```sql
INSERT INTO
  USERS (ID, NAME)
VALUES
  ('id', 'name') ON DUPLICATE KEY
UPDATE 
  NAME = VALUES(NAME)
```

**Expected Output**

```sql
INSERT INTO
  USERS (ID, NAME)
VALUES
  ('id', 'name') ON DUPLICATE KEY
UPDATE 
  NAME = VALUES(NAME)
```

**Actual Output**

```sql
INSERT INTO
  USERS (ID, NAME)
VALUES
  ('id', 'name') ON DUPLICATE KEY
UPDATE NAME =
VALUES
  (NAME)
```

**Usage**

- How are you calling / using the library?
  Use the js api.
- What SQL language(s) does this apply to?
Mysql
- Which SQL Formatter version are you using?
 Latest

**Comment by nene** [comment]
Thanks for reporting.

Unfortunately this is again one of those issues that demonstrates the fundamental problems with this formatter.

**Comment by nene** [comment]
I cobbled up a fix for this. Will get it into the next release...

**Comment by nene** [comment]
Published the fix in 13.0.0
